### PR TITLE
Milestone 2: Add API integration for my app

### PR DESCRIPTION
**Pull request by yonas44**
- Use fetch `POST` to send a request/ input score.
- Use fetch `GET` to receive a request with the scores.
- Use `async` and `await` to consume the API.

Thank you for reviewing my PR!

### CODE REVIEW

File: src/modules/pushTask.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable comma-dangle */
```
Comment (ha-manel) [inline src/modules/pushTask.js]: - [x] Kindly refrain from disabling linters (unless it is permitted in the project requirements), please remove this line and all the other lines that disable linters then work on fixing the errors. They're made so we work in a specific style and we should all respect that 💯

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @yonas44 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/pushTask.js (hunk 1 above)
Comment (yonas44) [reply src/modules/pushTask.js]: Okay, @ha-manel thank you for your feedback, I am getting this error because of prettier so I will have them removed.

**Review by victorokoroji — APPROVED** [review]
Hi @yonas44, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅
- Linters are working properly ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**